### ISSUE TITLE
Safe: status in core-p7 branch

### ISSUE THREAD

**Issue by jkeenan** [atoomic/perl#23]
In this ticket we will track our core-p7 progress on `dist/Safe`.

As of the morning of Jul 08 2020, we are getting this output from `make test` for this distro.

```
t/../dist/Safe/t/safe1 ............................................. Global symbol "%Config" requires explicit package name (did you forget to declare "my %Config"?) at t/safe1.t line 5.
Global symbol "%Config" requires explicit package name (did you forget to declare "my %Config"?) at t/safe1.t line 5.
BEGIN not safe after errors--compilation aborted at t/safe1.t line 10.
FAILED--no leader found
t/../dist/Safe/t/safe2 ............................................. Global symbol "%Config" requires explicit package name (did you forget to declare "my %Config"?) at t/safe2.t line 5.
Global symbol "%Config" requires explicit package name (did you forget to declare "my %Config"?) at t/safe2.t line 5.
BEGIN not safe after errors--compilation aborted at t/safe2.t line 9.
FAILED--no leader found
t/../dist/Safe/t/safe3 ............................................. Global symbol "%Config" requires explicit package name (did you forget to declare "my %Config"?) at t/safe3.t line 5.
Global symbol "%Config" requires explicit package name (did you forget to declare "my %Config"?) at t/safe3.t line 6.
Global symbol "%Config" requires explicit package name (did you forget to declare "my %Config"?) at t/safe3.t line 7.
BEGIN not safe after errors--compilation aborted at t/safe3.t line 12.
FAILED--no leader found
t/../dist/Safe/t/safeload .......................................... Global symbol "%Config" requires explicit package name (did you forget to declare "my %Config"?) at t/safeload.t line 6.
BEGIN not safe after errors--compilation aborted at t/safeload.t line 16.
FAILED--no leader found
t/../dist/Safe/t/safenamedcap ...................................... Global symbol "%Config" requires explicit package name (did you forget to declare "my %Config"?) at t/safenamedcap.t line 8.
BEGIN not safe after errors--compilation aborted at t/safenamedcap.t line 12.
FAILED--no leader found
t/../dist/Safe/t/safeops ........................................... Bareword found where operator expected at t/safeops.t line 49, near "new Safe"
	(Do you need to predeclare new?)
Array found where operator expected at t/safeops.t line 57, near "testop @"
	(Do you need to predeclare testop?)
syntax error at t/safeops.t line 49, near "new Safe"
Global symbol "$c" requires explicit package name (did you forget to declare "my $c"?) at t/safeops.t line 50.
syntax error at t/safeops.t line 57, near "testop @"
Execution of t/safeops.t aborted due to compilation errors.
FAILED--no leader found
t/../dist/Safe/t/saferegexp ........................................ Global symbol "%Config" requires explicit package name (did you forget to declare "my %Config"?) at t/saferegexp.t line 5.
BEGIN not safe after errors--compilation aborted at t/saferegexp.t line 9.
FAILED--no leader found
t/../dist/Safe/t/safesecurity ...................................... Global symbol "%Config" requires explicit package name (did you forget to declare "my %Config"?) at t/safesecurity.t line 6.
BEGIN not safe after errors--compilation aborted at t/safesecurity.t line 10.
FAILED--no leader found
t/../dist/Safe/t/safesort .......................................... Global symbol "%Config" requires explicit package name (did you forget to declare "my %Config"?) at t/safesort.t line 5.
Global symbol "%Config" requires explicit package name (did you forget to declare "my %Config"?) at t/safesort.t line 5.
BEGIN not safe after errors--compilation aborted at t/safesort.t line 9.
FAILED--no leader found
t/../dist/Safe/t/safeuniversal ..................................... Global symbol "%Config" requires explicit package name (did you forget to declare "my %Config"?) at t/safeuniversal.t line 6.
BEGIN not safe after errors--compilation aborted at t/safeuniversal.t line 10.
FAILED--no leader found
t/../dist/Safe/t/safeutf8 .......................................... Global symbol "%Config" requires explicit package name (did you forget to declare "my %Config"?) at t/safeutf8.t line 5.
Global symbol "%Config" requires explicit package name (did you forget to declare "my %Config"?) at t/safeutf8.t line 5.
BEGIN not safe after errors--compilation aborted at t/safeutf8.t line 9.
FAILED--no leader found
t/../dist/Safe/t/safewrap .......................................... Global symbol "%Config" requires explicit package name (did you forget to declare "my %Config"?) at t/safewrap.t line 6.
Global symbol "%Config" requires explicit package name (did you forget to declare "my %Config"?) at t/safewrap.t line 6.
BEGIN not safe after errors--compilation aborted at t/safewrap.t line 10.
FAILED--no leader found


```
These results were generated on an unthreaded build on Linux.  The results might differ significantly if run on a threaded build.  There have been subsequent commits to this branch which may have impacted these results.  YMMV.

**Comment by atoomic** [comment]
all the errors noticed here are fixed by 2018bb7b03911d963df4b3c4b8cc893cfe24fadb
we should be able to close this case

**Comment by jkeenan** [comment]
> all the errors noticed here are fixed by [2018bb7](https://github.com/atoomic/perl/commit/2018bb7b03911d963df4b3c4b8cc893cfe24fadb)
> we should be able to close this case

Yes, as of v5.32.0-576-g9e5ecda8ae, Safe is testing clean.  Thanks for your quick action.
